pilot/pkg/bootstrap/server.go

```diff
@@ -479,6 +494,9 @@ func (s *Server) initDiscoveryService(args *PilotArgs) error {
 	})
 
 	s.initGrpcServer(args.KeepaliveOptions)
+	if args.ServerOptions.GRPCAddr == "OFF" {
```

Comment:
nit: why not `""`? `"OFF"` seems a non standard

Reply:
Seemed a bit more readable/clear to me ? I don't care much, can change.

Reply:
We had many bugs around trying to do boolean matching on hardcoded strings in the past - led to an outage for one user actually which is the main concern.